swaybar/input.c

```diff
@@ -67,9 +67,19 @@ void update_cursor(struct swaybar_seat *seat) {
 	if (pointer->cursor_theme) {
 		wl_cursor_theme_destroy(pointer->cursor_theme);
 	}
+	const char *cursor_theme = getenv("XCURSOR_THEME");
+	unsigned cursor_size = 24;
+	const char *env_cursor_size = getenv("XCURSOR_SIZE");
+	if (env_cursor_size) {
+		char *end;
+		unsigned size = strtoul(env_cursor_size, &end, 10);
+		if (!*end) {
```

Comment:
Also check errno, and also check the string isn't empty.

Reply:
Fixed. I did **not** add a check for the empty string for `XCURSOR_THEME` as I consider `""` to be just like misspelling a valid theme name. If you think this is incorrect I'll of course add a check there too,